vhdbuilder/packer/generate-windows-vhd-configuration.ps1

```diff
@@ -192,10 +192,8 @@ $global:map = @{
         "https://acs-mirror.azureedge.net/kubernetes/v1.23.7/windowszip/v1.23.7-1int.zip",
         "https://acs-mirror.azureedge.net/kubernetes/v1.23.8-hotfix.20220728/windowszip/v1.23.8-hotfix.20220728-1int.zip",
         "https://acs-mirror.azureedge.net/kubernetes/v1.23.12-hotfix.20220922/windowszip/v1.23.12-hotfix.20220922-1int.zip", 
-        "https://acs-mirror.azureedge.net/kubernetes/v1.24.0/windowszip/v1.24.0-1int.zip",
-        "https://acs-mirror.azureedge.net/kubernetes/v1.24.3/windowszip/v1.24.3-1int.zip",
-        "https://acs-mirror.azureedge.net/kubernetes/v1.24.6/windowszip/v1.24.6-1int.zip",
-        "https://acs-mirror.azureedge.net/kubernetes/v1.25.2/windowszip/v1.25.2-1int.zip"
+        "https://acs-mirror.azureedge.net/kubernetes/v1.24.3-hotfix.20221006/windowszip/v1.24.3-hotfix.20221006-1int.zip", 
+        "https://acs-mirror.azureedge.net/kubernetes/v1.24.6-hotfix.20221006/windowszip/v1.24.6-hotfix.20221006-1int.zip"
     );
```

Comment:
Would it also be possible to make the corresponding version updates for Linux, like here: https://github.com/Azure/AgentBaker/pull/2230/files? While the contents would be the same for k8s binaries on Linux, it would be good to have parity for hotfix versions (before the auto patching pipeline is run), as we may also need to make corresponding changes on the RP side. Thank you.

Reply:
@ganeshkumar5699 This hotfix only has the hotfix for Windows so I think that Linux does not need to update the k8s packages. If we cache these packages for Linux, it means that we need a change in RP code. Will you do it?

Reply:
@ganeshkumar5699 Added the hotfix for 1.25.2 on Linux

Reply:
Thank you!